flexible-classroom/reference/classroom-sdk.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Classroom SDK'
```

Comment:
Suggestion: Rename this title to "Classroom SDK Client API". 
Reason: If we use the current title, such as "Classroom SDK" and "Classroom API", it's hard for users to understand and know their differences. So I think we can use a clear name instead.

Reply:
We all know SDK means the Client API. However, not all of us can realize that "API" means "REST API". 
Because Flexible Classroom has three kinds of APIs, I think it's better to use clear titles for these API docs, instead of hoping readers can know all things. Classroom Client API, Edu Context Client API, and Classroom Cloud Service RESTful API are highly recommended titles.

Reply:
:-), ok. I chatted with Max and agree that REST API for REST apis and SDK for sdk's :-). Happy compromise?